webapps/docs/config/filter.xml

```diff
@@ -319,6 +326,34 @@
         of <code>java.security.SecureRandom</code> will be used.</p>
       </attribute>
 
+      <attribute name="noNonceURLPatterns" required="false">
+        <p>A list of URL patterns that will <i>not</i> have CSRF nonces added
+        to them. You may not want to add nonces to certain URLs to avoid
+        creating unique URLs which may defeat resource caching, etc.</p>
+
+        <p>There are 3 types of patterns supported:</p>
```

Comment:
3 == three

Though there are actually 4 (four?), now, so that needs a fix.

Is there a documented preference for spelled-out numerals in Tomcat documentation?

Reply:
Most style guides will likely tell your to which number should be written up. APA, Chicago, etc.

Reply:
2ab2317d581fb6657fe02529d5ad55af00161726